recipes/pom.xml

```diff
@@ -117,6 +124,58 @@
             <artifactId>rewrite-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.4.0</version>
```

Comment:
Use Maven properties for all versions if possible. It'll make maintenance simpler